core/src/avm2/globals/toplevel.rs

```diff
@@ -36,184 +35,33 @@ pub fn trace<'gc>(
     Ok(Value::Undefined)
 }
 
-pub fn log_warn<'gc>(
```

Comment:
can we keep `log_warn`? It's sometimes useful as a louder-than-`trace` logging solution for SWF debugging.